### PR TITLE
ENH: Pretrain Actor-Critic

### PR DESCRIPTION
**Pull request by AntoineTheb**

### CODE REVIEW

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+where $$w, \theta$$ are the parameters for the critic and actor networks. We can then combine them to the standard DDPG gradients 
+
+![](/article/images/pretrainac/grads.jpeg)
+
+TLDR: Calculate the advantage of the expert policy. Multiply the advantage to the gradients of the weights of the critic. Add it to the gradient from the raw algorithm.
+
+## Data
+
+The tests we made on the HalfCheetah, Hopper, and Walker2d OpenAI Gym environments.
```
Comment (jhlegarreta) [inline article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: *Tests were one on the (...)*
or
*Authors tested their method on the (...)*
?

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+layout: review
+title: "Pretraining Deep Actor-Critic Reinforcement Learning Algorithms With Expert Demonstrations"
+tags: deep-learning reinforcement supervised
+author: ""
```
Comment (jhlegarreta) [inline article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: Your name is missing here.

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+layout: review
+title: "Pretraining Deep Actor-Critic Reinforcement Learning Algorithms With Expert Demonstrations"
+tags: deep-learning reinforcement supervised
```
Comment (jodoin) [inline article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: supervised is not a valid tag

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+$$\eta(\pi^*) \geq \eta(\pi)$$
+
+which we will call constraint $$(1)$$
+
+The authors then introduce the following theorem and "prove it":
+
+![](/article/images/pretrainac/theorem1.jpeg)
+
+which gives us constraint $$(3)$$
```
Comment (jodoin) [inline article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: which gives the constraint

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: Done in e263bbb.

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: Done in e263bbb.

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md (hunk 3 above)
Comment (jhlegarreta) [reply article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: Removed in e263bbb.

File: article/_posts/2019-06-05-Pretrain_Actor_Critic.md (hunk 4 above)
Comment (jhlegarreta) [reply article/_posts/2019-06-05-Pretrain_Actor_Critic.md]: Done in e263bbb.

**Pull request merged**